app/theme.scss

```diff
@@ -387,3 +387,28 @@ input.react-tags__combobox-input:focus {
 .usa-input-prefix.wide-input-prefix + * {
   padding-left: units(10);
 }
+
+.usa-prose--dark {
+  @include u-bg('ink');
+  @include u-text('white');
+}
+
+/*
+ * Workaround for https://github.com/uswds/uswds/issues/5695
+ */
+figure {
+  & > figcaption {
+    @include u-text('gray-30');
+    margin-left: 2rem;
+    font-style: italic;
+  }
+  & > figcaption > img {
+    padding-bottom: 1rem;
+    padding-right: 0.5rem;
+    float: left;
+  }
+}
+figure.bordered {
+  @include u-border-left('violet-70v');
```

Comment:
How about one of the the theme accent colors?

Reply:
Yes, please try one of the named theme colors.

Reply:
ok. I think the purple is better looking. I'm changing it to primary as it's the least ugly color, but the blue is different than nasa blue so they kinda clash a bit with the logos.

Reply:
<img width="900" alt="Screenshot 2024-01-02 at 1 34 28 PM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/c697ffe8-08d2-473f-93be-4322200d01b8">

Reply:
How about `text-base`?

Reply:
I believe you can't use text-base as a border color because borders aren't text.

Reply:
Right, so how about `border-base`?

Reply:
There is not a border-base, but I can do just plain base.

Reply:
There is a CSS utility class called `border-base`, but when defining a CSS rule you would use the following:

```
@include u-border-left(3px, 'base');
```

Reply:
that's what I did.

Reply:
except it broke if I passed in the border weight so I put that separately. I have to dig a little because according to the documentation it should work with the weight and the color.

Reply:
so it broke because I made it 3 px and there is not a valid option for 3px passed in (which is kind of stupid). so I can make it 3 pixels wide if I declare the weight separately or I can use the utility with both passed in and the border thinner.

Reply:
That's super weird. I don't know why that would be.

You could tie this to a theme prop by using `$theme-card-border-width` or something similar.